pyproject.toml

```diff
@@ -67,6 +68,7 @@ protobuf = "^4.25.2"
 cryptography = "^41.0.2"
 pycryptodome = "^3.18.0"
 iterators = "^0.0.2"
+typer = { version = "^0.9.0", extras=["all"] }
```

Comment:
Do we need `all` extras, or can we depend on fewer packages?

Reply:
The dependencies installed with all are:

Link: https://github.com/tiangolo/typer/blob/3a7264cd56181690805f220cb44a0301c0fdf9f3/pyproject.toml#L67-L71

```toml
all = [
    "colorama >=0.4.3,<0.5.0",
    "shellingham >=1.3.0,<2.0.0",
    "rich >=10.11.0,<14.0.0",
]
```

We need `colorama` and `shellingham` and might use features exposed with `rich` (mostly output formatting) in the future.

So I'd suggest to keep the "all"